requirements.txt

```diff
@@ -5,3 +5,4 @@ docopt
 requests_cache
 pytablewriter
 publicsuffix
+pyopenssl=17.2.0
```

Comment:
Is there a reason that pyopenssl is oinned but others are not? If so, should we document that here?